client/src/components/ManageAccount/ManageAccount.js

```diff
@@ -1,9 +1,9 @@
 import React, { useEffect } from 'react';
```

Comment:
__Out of Scope__
I did **not** convert Manage Account. Its layout was _too complex_, and I did not have skill yet to do it. Dashboard was the closest applicable layout, but the layout code for Dashboard is barely maintainable (again, past skill limitation), and should not be re-used. I have the skill to do it, now, but not the time.

Reply:
__Fixed__

- 444b879 I added brittle code to avoid the cut off for the screen width range in which it happens.
    - [Manage Account License Buttons 991—1200](https://user-images.githubusercontent.com/62723358/116608890-53bab480-a8f9-11eb-8070-16f7cd3da5fc.png)
    - [Manage Account  1200](https://user-images.githubusercontent.com/62723358/116608894-54534b00-a8f9-11eb-978e-1a3415e2018f.png)
    - [Manage Account License Buttons  991](https://user-images.githubusercontent.com/62723358/116608896-54ebe180-a8f9-11eb-9956-44585772ce58.png)

Reply:
__Reverted via dbb4d23__
I broke a unit test. The fix was not quick. I didn't like the solution anyway. We both think this is out of scope.

_**But**, the Manage Account layout does look broken compared to before (content escapes horizontal "edges" of section. **So**, I am adding a task to fix that._

Reply:
>  I am adding a task to fix that.
awesome 👍

Reply:
__Fixed, But Reverted__
Solution implemented, then reverted. Broke a unit test I don't know how to appropriately fix, and the solution was brittle and out-of-scope. But, the Manage Account page layout is actually broken compared to `main`, so I must fix that. PR task added.

Reply:
__Fixed (Again)__
I changed the text to match design. This text is short enough to not be cut off.

- 9ea8c7b
    - [Account (Wide) 992x485](https://user-images.githubusercontent.com/62723358/116754037-b54c5300-a9cd-11eb-8c8d-72d9217de5e1.png)
    - [Account (Narrow) 991x485](https://user-images.githubusercontent.com/62723358/116754045-b67d8000-a9cd-11eb-97d8-cc6d40d1cfc5.png)

Reply:
__Update__
I updated `<ManageAccount>` to use new layout for multi-column unequal widths (updated `<Dashboard>` also) via 6136a57 then 1bcba4e.